.travis.yml

```diff
@@ -6,7 +6,12 @@ install:
 
 matrix:
   include:
-    - env: JOB_TYPE=dashing
+    # - env: JOB_TYPE=crystal
+    #   script: .ros2ci/travis.bash $JOB_TYPE
+    # Uncomment the following to test against ROS 2 Dashing
+    # - env: JOB_TYPE=dashing
+    #   script: .ros2ci/travis.bash $JOB_TYPE
```

Comment:
why removing dashing?

Reply:
This was initially targeting eloquent, but I think it makes more sense to use nightly. (Unless there's another reason we want to target dashing).